Taskfile.Shared.yml

```
@@ -62,56 +62,39 @@ tasks:
     label: "{{.TASK}}-{{.NAME}}"
     dir: coprocessors/{{.NAME}}
     cmds:
-      - docker run -d --rm --name {{.NAME}}-co --env PORT="{{.COPROCESSOR_PORT}}" -m 1g --cpus=1 -p {{.COPROCESSOR_PORT}}:{{.COPROCESSOR_PORT}} {{.NAME}}-co
+      - docker run -d --rm --name coprocessor --net=task --env PORT="8000" -p 8000:8000 -m 1g --cpus=1 {{.NAME}}-co
     vars:
       NAME: "{{.NAME|default .CLI_ARGS}}"
   cleanup-coprocessor:
     label: "{{.TASK}}-{{.NAME}}"
     cmds:
-      - docker stop {{.NAME}}-co
+      - docker stop coprocessor
     vars:
       NAME: "{{.NAME|default .CLI_ARGS}}"
 
   start-subgraphs:
     cmds:
-      - task: build-subgraph
-        vars:
-          NAME: a
-      - task: build-subgraph
-        vars:
-          NAME: b
-      - task: start-subgraph
-        vars:
-          NAME: a
-          PORT: 8081
-      - task: start-subgraph
-        vars:
-          NAME: b
-          PORT: 8080
-  cleanup-subgraphs:
-    cmds:
-      - task: cleanup-subgraph
-        vars:
-          NAME: a
-      - task: cleanup-subgraph
-        vars:
-          NAME: b
-
-  build-subgraph:
-    internal: true
-    label: "build-subgraph-{{.NAME}}"
-    dir: subgraphs/subgraph-{{.NAME}}
+      - task: build-fixed-subgraph
+      - task: start-fixed-subgraph
+  build-fixed-subgraph:
+    dir: subgraphs/fixed
     cmds:
-      - docker build . -t sub-{{.NAME}}
+      - DOCKER_BUILDKIT=1 docker build . -t fixed
     sources:
       - "*.*"
-  start-subgraph:
-    internal: true
-    label: "start-subgraph-{{.NAME}}"
+  start-fixed-subgraph:
+    dir: subgraphs/fixed
     cmds:
-      - docker run --rm -d --name sub-{{.NAME}} --env PORT="{{.PORT}}" -m 1g --cpus=1 -p {{.PORT}}:{{.PORT}} sub-{{.NAME}}
-  cleanup-subgraph:
-    internal: true
-    label: "cleanup-subgraph-{{.NAME}}"
+      - docker run --rm -d --name fixed -p 8082:8082 fixed
```

```suggestion
      - docker run --rm -d --name subgraph --net=task -p 8082:8082 fixed
```

errors due to the router being unable to reach it due to missing net/mismatched name

---

Oh interesting—this passed in GHA but I didn't look at the results 😆. If we do much iterating in the future, we should probably make the tasks fail (product non-0 exit code) if we get all errors from the router.